content/applications/general/email_communication/faq.rst

```diff
@@ -1,173 +1,232 @@
-  `Odoo Support <https://www.odoo.com/help>`_ , if you have a recurring issue
-  with the same client or domain.
+When there is an issue with incoming emails, there might not be an indication, per se, in Odoo. It
+is the sending email client, who tries to contact a database, that will get a bounce-back message
+(most of the time a :guilabel:`550: mailbox unavailable` error message).
 
-Get help from support
+Email is not received
 ---------------------
 
-In order to get helped efficiently, please provide as much information as possible. Here is a list
-of what can be helpful:
+The steps that should be taken depend on the Odoo platform where the database is hosted.
 
-- The **EML** of the email, stating for *Electronic Mail*, is the file format containing all the
-  technical information required for an investigation. The documentation of your own email provider
-  might help you on how to get your EML files. Once you get the EML of the email, adding it
-  in the attachment of your ticket is the most efficient way for us to investigate. The support
-  will mainly focus on redundant issues.
+**Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
 
-  .. seealso::
-     - `Gmail documentation
-       <https://support.google.com/mail/answer/29436>`_
-     - `Outlook documentation
-       <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
+.. tip::
+   Logs are a stored collection of all the tasks completed in a database. They are a text only
+   representation, complete with timestamps of every action taken on the Odoo database. This can be
+   helpful to track emails leaving the database. Failure to send can also be seen by logs that
+   indicate that the message tried to send repeatedly. Logs will show every action to the email
+   servers from the database.
 
-- The exact flow you are following in order to normally receive those emails in Odoo. Here are
-  examples of questions whose answers can be useful:
+   The folder :file:`~/logs/` (accessed by the command line or on the Odoo.sh dashboard) of an
+   Odoo.sh database contains a list of files containing the logs of the database. The log files are
+   created everyday at 5:00 AM (UTC).
 
-  - Is this simply a reply from an email going out from Odoo ?
+   The two most recent days (today and yesterday) are not compressed, while the older ones are, in
+   order to save space. The naming of the files for today and yesterday are respectively:
+   :file:`odoo.log` and :file:`odoo.log.1`.
 
-  - Are you using an incoming email server or somehow redirecting?
+   For the following days, they are named with their dates, and then compressed. Use the command
+   :command:`grep` and :command:`zgrep` (for the compressed ones) to search through the files.
 
-  - Can you provide us with an example of an email that has been correctly forwarded ?
+   .. seealso::
```

Comment:
This 'See also' box could definitely stand on its own instead of being inside the Tip box